kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/connector/ConnectorsList.java

```diff
@@ -50,11 +51,11 @@ public ConnectorsList isNotVisible(String connectorName) {
     }
 
     @Step("Verify that connector {connectorName} is visible in the list")
```

Comment:
step description is not actual

Reply:
Fixed

Reply:
can't see anyy changes here. where did you fix it??

Reply:
can't see any changes here. where did you fix it??

Reply:
can't see any changes here. where did you fix it??